documentation/operations-manual.md

```diff
@@ -1 +1,36 @@
-# Travelcy Operations Manual
\ No newline at end of file
+# Travelcy Operations Manual
+
+## Overview
+Travelcy is a native android application developed in Kotlin with Android Studio.
+
+## Coding conventions
```

Comment:
I feel like we are missing a more in depth description on how the app is implemented and how things work together, the information needed for a developer that we would handover the project to and would need to implement a new feature.